### PR TITLE
MF-571 - UI Thing module with modals

### PR DESCRIPTION
**Pull request by darkodraskovic**
Add complete thing manipulation via modals.

### CODE REVIEW

**Review by blokovi — APPROVED** [review]
Great work @darkodraskovic 
LGTM

File: ui/src/Thing.elm [hunk 1]
```diff
@@ -11,9 +13,9 @@ import Bootstrap.Modal as Modal
 import Bootstrap.Table as Table
 import Bootstrap.Utilities.Spacing as Spacing
 import Debug exposing (log)
-import Dict exposing (Dict)
+import Dict as Dict
```
Comment (anovakovic01) [inline ui/src/Thing.elm]: @darkodraskovic Why don't you just use `import Dict`? It will have the same effect, right?

Comment (darkodraskovic) [reply ui/src/Thing.elm]: Fixed in the last commit.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**